babel/messages/pofile.py

```diff
@@ -521,14 +522,21 @@ def _write(text):
             text = text.encode(catalog.charset, 'backslashreplace')
         fileobj.write(text)
 
+<<<<<<< HEAD
+    def _write_comment(comment, prefix='':
+=======
```

Comment:
Merge conflict marker and syntax error.

Reply:
I don't know why there is an error here, the code at my local appeared to be OK. I will make another push to ensure the correct code is placed.